kani-compiler/src/codegen_cprover_gotoc/codegen/rvalue.rs

```diff
@@ -844,14 +841,11 @@ impl<'tcx> GotocCtx<'tcx> {
                 format!("drop_unimplemented<{}>", self.readable_instance_name(drop_instance));
             let drop_sym = self.ensure(&drop_sym_name, |ctx, name| {
                 // Function body
-                let unimplemented = ctx
-                    .codegen_unimplemented(
-                        format!("drop_in_place for {}", drop_sym_name).as_str(),
-                        Type::empty(),
-                        Location::none(),
-                        "https://github.com/model-checking/kani/issues/281",
-                    )
-                    .as_stmt(Location::none());
+                let unimplemented = ctx.codegen_unimplemented_stmt(
+                    format!("drop_in_place for {}", drop_sym_name).as_str(),
+                    Location::none(),
```

Comment:
No location to use here?

Reply:
I'm fixing missing locations in some of the statement stuff, but apparently here is vtable generation we don't really have a location?